### PR TITLE
Fix death graph, add zoom, add axes units

### PR DESCRIPTION
**Pull request by JaafarRammal**
# Summary

- Fix agent type deaths
- Add zoom
- Add units to axis

### CODE REVIEW

File: pkg/utils/logging/statelog.go [hunk 1]
```diff
@@ -11,6 +11,7 @@ import (
 )
 
 type msgMap [14][8]int
+type deathMap [8]int
 type treatyResponsesCount [2][8]int // 0 for reject, 1 for accept
```
Comment (jzzheng22) [inline pkg/utils/logging/statelog.go:15]: hate this lmao but fine given deadline i suppose

are we fixing the 14 to 12?

**Review by jzzheng22 — COMMENTED** [review]
Zooming also zooms the vertical axis which is a little annoying - I feel like most of the time I'd only want to zoom in horizontal axis and not vertical.

**Comment by jzzheng22** [comment]
(as usual, not reviewing the typescript)

File: pkg/utils/logging/statelog.go (hunk 1 above)
Comment (JaafarRammal) [reply pkg/utils/logging/statelog.go:15]: uhh 14 to 12 is Daryl's code but it works so we didn't want to mess with it :))

Comment (JaafarRammal) [reply pkg/utils/logging/statelog.go:15]: uhhh 14 to 12 works fine for now @daryllimyt and I didn't want to break something with it
are message types unused still showing in graphs?

Comment (jzzheng22) [reply pkg/utils/logging/statelog.go:15]: Yes, that PR hasn't been merged

Comment (JaafarRammal) [reply pkg/utils/logging/statelog.go:15]: ohh, will look at it

**Comment by udai-arneja** [comment]
Thanks ! Looks good to me & intuitive to use :)

**Review by udai-arneja — APPROVED** [review]

File: frontend/src/Components/Results/Graphs/MultiScatterChart.tsx [hunk 2]
```diff
@@ -34,11 +40,70 @@ export default function MultiScatterChart(props: MultiScatterChartProps) {
...
+                    enabled: true,
+                  },
+                  mode: "xy",
+                },
+              },
+            },
+            scales: {
+              y: {
+                ticks: {
+                  // Include a dollar sign in the ticks
```
Comment (JaafarRammal) [inline frontend/src/Components/Results/Graphs/MultiScatterChart.tsx:88]: butchering my own code what is this...

Comment (JaafarRammal) [reply frontend/src/Components/Results/Graphs/MultiScatterChart.tsx:88]: needs some cleanup

**Review by udai-arneja — DISMISSED** [review]

**Review by udai-arneja — APPROVED** [review]

File: frontend/src/index.css [hunk 3]
```diff
@@ -48,4 +48,12 @@ body {
...
   animation: rotation 2s;
+}
+
+.col-lg-6{
+  padding: 10px !important;
+}
+
+.col-lg-12{
+  padding: 10px !important;
 }
```
Comment (daryllimyt) [inline frontend/src/index.css:59]: missing newline?

**Review by daryllimyt — APPROVED** [review]
LGTM

File: frontend/src/index.css (hunk 3 above)
Comment (JaafarRammal) [reply frontend/src/index.css:59]: I think it's fine! linter is happy

**Pull request merged**